tools/objc/BUILD

```diff
@@ -16,9 +16,15 @@ sh_binary(
     data = [
         ":make_hashed_objlist.py",
         ":xcrunwrapper",
+        ":libtool_check_unique",
     ],
 )
 
+cc_binary(
+    name = "libtool_check_unique",
+    srcs = ["libtool_check_unique.cc"],
```

Comment:
I can't figure out how this stuff is actually used, so I can't tell if it works or not.

Reply:
Doing some archaeology, I think this was a remnant of the pre-crosstool ObjcCompilation rule implementations. Which is presumably why you can't find it. 

I bet this can actually be removed, but I am not knowledgeable enough about the Bazel CI to know if we'd catch any issues.

Reply:
related https://github.com/bazelbuild/bazel/pull/12073/

I do think some of it looks more used in j2objc stuff tho

Reply:
It looks like they pass it along in the J2ObjcAspect, but I can't find any references to it being used after that. Did you catch something else? (We can also take this to your other PR)

Reply:
sounds like we can pretty safely ignore this piece for this use case and fix it if anything comes up, so I think this PR should be good to go!

Reply:
so then i can leave it as-is and we can merge? or does it need to be deleted?

Reply:
I would delete it for now unless we can prove it is needed otherwise.

Reply:
Up to whoever can merge, but unless mine merges first I do think this should work, so we could leave it to try and get this in 3.6, and I can remove it if it's really unused.